lib/evss/disability_compensation_form/form526_to_lighthouse_transform.rb

```diff
@@ -313,14 +315,18 @@ def transform_multiple_exposures(details, multiple_exposures_type = MULTIPLE_EXP
         end
       end
 
-      def transform_multiple_exposures_other_details(details)
+      def transform_multiple_exposures_other_details(details, multiple_exposures_type)
         obj = Requests::MultipleExposures.new(
           exposure_dates: Requests::Dates.new
         )
 
         obj.exposure_dates.begin_date = convert_date_no_day(details['startDate']) if details['startDate'].present?
         obj.exposure_dates.end_date = convert_date_no_day(details['endDate']) if details['endDate'].present?
-        obj.exposure_location = details['description'] if details['description'].present?
+        if multiple_exposures_type == MULTIPLE_EXPOSURES_TYPE[:hazard]
+          obj.hazard_exposed_to = details['description'] if details['description'].present?
+        elsif details['description'].present?
+          obj.exposure_location = details['description']
+        end
```

Comment:
Redundancy can be reduced by first checking the presence of `details['description']` and then handling the assignment based on the value of `multiple_exposures_type`. For example:
```
if details['description'].present?
  if multiple_exposures_type == MULTIPLE_EXPOSURES_TYPE[:hazard]
    obj.hazard_exposed_to = details['description']
  else
    obj.exposure_location = details['description']
  end
end
```

Reply:
You're right, this would've been much cleaner, I'll revisit this on my next PR. Rubocop actually rewrote whatever I originally had into the form you now see, but it seems to like your version without change 😂